.github/workflows/test.yml

```diff
@@ -0,0 +1,70 @@
+# This workflow will install Python dependencies, run tests and lint
+# with a variety of Python versions.  For more information see:
+# https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+    
+jobs:
+  test:
+
+    runs-on:
+      - ubuntu-latest
```

Comment:
Should we use a fixed Ubuntu version? Maybe the latest LTS version?

Reply:
That is a good consideration.  GitHub might upgrade the runner images and break CI for us.  In practice I have not found this to be a big problem though.  Notes on https://github.com/actions/runner-images says that

> Before moving the-latest label to a new OS version we will announce the change and give sufficient lead time for users to update their workflows

I think keeping up with `-latest` will allow us to keep up with somewhat recent but not too old platforms.  If we use `ubuntu-20.04`, we might fall behind in no time.  We might not be ready to use `ubuntu-22.04` right now, especially considering that we tend to use older and stabler platforms on our test beds.

What do you think?